### PR TITLE
Rdfe publisher apis

### PR DESCRIPTION
**Pull request by julla**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@julla__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: src/StorSimple/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure StorSimple management functions")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.2.0")]
```
Comment (huangpf) [inline src/StorSimple/Properties/AssemblyInfo.cs]: Please revert this change.

File: src/SiteRecoveryManagement/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -20,7 +20,7 @@
 [assembly: AssemblyDescription("")]
 
 [assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.5.1.0")]
+[assembly: AssemblyFileVersion("0.6.0.0")]
```
Comment (huangpf) [inline src/SiteRecoveryManagement/Properties/AssemblyInfo.cs]: Please revert this change.

File: src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj [hunk 3]
```diff
@@ -5,7 +5,7 @@
     Microsoft.WindowsAzure.Management.Compute
     -->
     <SdkNuGetPackage Include="Microsoft.WindowsAzure.Management.Compute">
-      <PackageVersion>11.2.1</PackageVersion>
+      <PackageVersion>11.2.2</PackageVersion>
```
Comment (huangpf) [inline src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj]: If it's not a breaking change, and you are adding a new API, it shall be ```11.3.0```.

@markcowl Could you please help check?

**Comment by huangpf** [comment]
this this please

File: src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj (hunk 3 above)
Comment (julla) [inline src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj]: It is a breaking change given the new data model contract changes in Replicate request payload and GetDetails response payload.

Comment (huangpf) [inline src/ComputeManagement/Microsoft.WindowsAzure.Management.Compute.nuget.proj]: Then it should be ```12.0.0```.

**Comment by julla** [comment]
Duplicate of https://github.com/Azure/azure-sdk-for-net/pull/1050

Will close.
